### PR TITLE
Update ProductDescription README.md

### PR DESCRIPTION
**Pull request by JuanAlmeida12**
#### What is the purpose of this pull request?
As title says.

#### Types of changes
- [ ] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/components/ProductDescription/README.md [hunk 1]
```diff
@@ -1,24 +1,44 @@
 # Product Description
-Product Description is a canonical component that any VTEX app can import.
 
-To import it into your code: 
-```js
-import ProductDescription from 'vtex.store-components/ProductDescription'
-```
+## Description
+
+`ProductDescription` is a VTEX Component that shows the description of the product.
```
Comment (luciannojunior) [inline react/components/ProductDescription/README.md]: a 
```suggestion
`ProductDescription` is a VTEX Component that shows the description of a product.
```

Comment (theomoura) [inline react/components/ProductDescription/README.md]: Shouldn't be: 

`...of a product`?

File: react/components/ProductDescription/README.md [hunk 2]
```diff
@@ -1,24 +1,44 @@
...
+This Component can be imported and used by any VTEX App.
+
+:loudspeaker: **Disclaimer:** Don't fork this project, use, contribute, or open issue with your feature request.
```
Comment (Klynger) [inline react/components/ProductDescription/README.md]: Shouldn't this be only in the main `README` of `store-components`?

File: react/components/ProductDescription/README.md [hunk 3]
```diff
@@ -1,24 +1,44 @@
...
 ## Usage
-You can use it in your code like a React component with the jsx tag: `<ProductDescription />`. 
-```jsx
-<ProductDescription specifications={product.specs}>
-   <span>{product.description}</span>
-</ProductDescription>
+
+You should follow the usage instruction in the main [README](https://github.com/vtex-apps/store-components/blob/master/README.md#usage).
+
+Then, add `product-description` block into our app theme, as we do in our [Product Details app](https://github.com/vtex-apps/product-details/blob/master/store/blocks.json).
```
Comment (luciannojunior) [inline react/components/ProductDescription/README.md]:
```suggestion
Then, add `product-description` block into your app theme, as we do in our [Product Details app](https://github.com/vtex-apps/product-details/blob/master/store/blocks.json). 
```

**Comment by GustavoMA** [comment]
Changelog

File: react/components/ProductDescription/README.md (hunk 2 above)
Comment (GustavoMA) [reply react/components/ProductDescription/README.md]: There is no problem in also putting it here IMO

File: CHANGELOG.md [hunk 4]
```diff
@@ -6,6 +6,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
```
Comment (GustavoMA) [inline CHANGELOG.md]: There is already a Added Section below, use it.

File: README.md [hunk 5]
```diff
@@ -83,6 +84,21 @@ export default ProductPrice
 
 Also, all dependencies needed should be inserted inside the react/package.json
 
+## Styles API
```
Comment (GustavoMA) [inline README.md]: Add this section in `Usage` as a subsection.

File: README.md [hunk 6]
```diff
@@ -83,6 +84,21 @@ export default ProductPrice
...
+This app has CSS customization through `CSS Modules`. CSS Modules is a CSS file in which all class names and animation names are scoped locally by default. You can read more about CSS Modules [here](https://github.com/css-modules/css-modules).
+
+We use it `css-loader` to generate a CSS token on a HTML element. For example, the builder generate a CSS token based on app vendor, name and major version. Like `productNameContainer` token declared in `ProductName`, generate the classname `vtex.store-components-3-x-productNameContainer`.
+
```
Comment (GustavoMA) [inline README.md]: Add a text explaining that each Component has its own README with the tokens described.

**Pull request merged**